### DISCUSSION TITLE
Open application with backstage completely open

### DISCUSSION THREAD

**Discussion by TAlecksen** [fluentribbon/Fluent.Ribbon#1223] [General]
Is it possible to make the backstage window be open when the application starts? I am aware of the setting

`ribbonBackstage.IsOpen = true;`

but this makes the backstage open AFTER everything is loaded which causes some strange visualizations depending what is on the main window.  I also tried putting the IsOpen inside the XAML but it still seems to load the same way. Lastly, I tried using a StartScreen but this also acts the same.

Any help would be appreciated on how to do this, if it is even possible.

**Comment by batzen** [comment]
Will have a look on Monday.
Do you have a repro? It seems to be dependent on what's shown in the window, or does it repro with the showcase application?

**Reply by TAlecksen** [reply]
Thank you, it does it regardless of what is in the main Window. I have just observed that if the main window contains something that has a lot of colors it looks strange when opening since it flashes the window briefly before opening the backstage.

**Reply by batzen** [reply]
This is how it looks, slowed down, with the showcase application:
![Ribbon_#1223](https://github.com/user-attachments/assets/9514876a-eb41-4f10-8c99-82fd9ffac1d3)

If your content show before the start screen is shown there might some load-order issue in your app.
What you could do is hide your content till the start screen is shown.

**Reply by TAlecksen** [reply]
I must be doing something wrong. Here's what I am seeing when using the sample code and putting a circle on the main form.
 
https://github.com/user-attachments/assets/da2da205-44eb-4995-8779-ab873eef3945

When you use the backstage instead of the start screen, does it still show the same on your side?

**Reply by batzen** [reply]
Using a backstage instead shouldn't change much.

It looks like a load order issue.
To fully prevent that from happening you should put your regular window content in a container (grid for example), set it's visibility to collapsed and only set it to visible when the start screen has been shown.
